.gitignore

```diff
@@ -0,0 +1 @@
+.vagrant
```

Comment:
@olivermel Unfortunately adding this to gitignore is necessary, but not enough to get rid of the existing commits.  You'll have to run `git rm --cached -r .vagrant', then git commit and git push again.  the '--cached' part tells Git to remove the files from the repository, but not delete them on your local machine.  Then, because of the .gitignore, they will never get added back again.